satellite/console/consoleweb/consoleql/query_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"storj.io/storj/satellite/console/consoleauth"
 	"storj.io/storj/satellite/console/consoleweb/consoleql"
 	"storj.io/storj/satellite/mailservice"
+	"storj.io/storj/satellite/payments/internalservice"
```

Comment:
can we name this package something more descriptive? `internalservice` is surprising